dev/io.openliberty.org.jboss.resteasy.mprestclient/bnd.bnd

```diff
@@ -52,6 +52,9 @@ Import-Package: \
   org.eclipse.microprofile.rest.client.*;version='!',\
   *
 
+DynamicImport-Package: \
+  com.ibm.ws.microprofile.faulttolerance.cdi
```

Comment:
It's necessary in order for the rest client implementation to add the Liberty-specific FT annotation class.  Since the actual "bean" is a proxy class, the existing FT code doesn't handle the annotations.  We did the same thing with MP Rest Client 1.X and 2.X.